systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -298,14 +291,14 @@ public void testCustomAndUpdatedValues() {
     }
 
     @Test
-    @JUnitGroup(name = "regression")
+//    @JUnitGroup(name = "regression")
```

Comment:
Is this `//` intentional? Or just something you forgot to delete before committing it?